lib/Zonemaster/CLI.pm

```diff
@@ -292,37 +307,47 @@ sub run {
         print_test_list();
     }
 
-    Zonemaster::Engine->config->ipv4_ok(0+$self->ipv4);
-    Zonemaster::Engine->config->ipv6_ok(0+$self->ipv6);
+    Zonemaster::Engine->profile->ipv4_ok(0+$self->ipv4);
+    Zonemaster::Engine->profile->ipv6_ok(0+$self->ipv6);
 
     if ($self->sourceaddr) {
-        Zonemaster::Engine->config->resolver_source($self->sourceaddr);
+        Zonemaster::Engine->profile->resolver_source($self->sourceaddr);
     }
 
     # Filehandle for diagnostics output
     my $fh_diag = ( $self->json or $self->json_stream or $self->raw or $self->dump_config or $self->dump_policy )
       ? *STDERR     # Structured output mode (e.g. JSON)
       : *STDOUT;    # Human readable output mode
 
-    if ( $self->policy ) {
-        say $fh_diag __x( "Loading policy from {path}.", path => $self->policy );
-        Zonemaster::Engine->config->load_policy_file( $self->policy );
+    if ( $self->profile ) {
+        say $fh_diag __x( "Loading profile from {path}.", path => $self->profile );
+        Zonemaster::Engine->profile->load( $self->profile );
     }
+    else {
 
-    if ( $self->config ) {
-        say $fh_diag __x( "Loading configuaration from {path}.", path => $self->config );
-        Zonemaster::Engine->config->load_config_file( $self->config );
+        if ( $self->policy ) {
+            say $fh_diag __x( "Loading policy from {path}.", path => $self->policy );
+            say $fh_diag __x( "DEPRECATED, use profile instead." );
```

Comment:
We should use a stronger word than deprecated if we make it an error to use it. Deprecation implies that the old way still works for the time being but will be removed in a future version.
https://en.wikipedia.org/wiki/Deprecation

Reply:
I suggest that we write "Use of config and policy have been TERMINATED. Use profile instead."

Reply:
Done this way.